src/main/java/org/openapitools/sdk/storage/Storage.java

```diff
@@ -87,7 +87,7 @@ public static Long getExpiredAt(HttpServletRequest request) {
     }
 
     public static Long getTokenTimeToLive() {
-        return tokenTimeToLive != null ? tokenTimeToLive : System.currentTimeMillis() + 3600 * 24 * 15 * 1000; // Live in 15 days
+        return tokenTimeToLive != null ? tokenTimeToLive : 3600 * 24 * 15; // Live in 15 days
```

Comment:
Can define as a `static final int FIFTEEN_DAYS = 3600 * 24 * 15` and use in getTokenTImeToLeave. And the function can also simplify:
```
public static Long getTokenTimeToLive() {
        return tokenTimeToLive == null ? FIFTEEN_DAYS : tokenTimeToLive;
}
```

Reply:
As you are calculating token time in seconds 15 days ahead from now. Why you removed `System.currentTimeMillis()`. It shouldn't be remove.

Final implementation should look like:

`public static Long getTokenTimeToLive() {
        return tokenTimeToLive != null ? tokenTimeToLive : (System.currentTimeMillis()/1000) + 3600 * 24 * 15; // Live in 15 days
        }
`

Also, you should use constants.

Reply:
@7171mayanktyagi that was removed in https://github.com/kinde-oss/kinde-java-sdk/pull/18/commits/f87edcb59806eccf101242d2c80666ca88738201 My comments was related to current PR, if calculation will be changed, could properlu change the above my comments where already constants assigned. Which was my only notes + simplify nullify check, and not negate null checking.

Reply:
As, value of getTokenTimeToLive() is being passed in setMaxAge() of Cookie. And it accept value in ms.

Reply:
@hhovhann I just verified in java documentation. @arnfred implementation is absolutely correct. Cookie 's maxage set in seconds.

Reply:
@hhovhann, I also came across this **#PR** on **Upwork**. The question is, who is the author of this post?

Reply:
https://github.com/kinde-oss/kinde-java-sdk/pull/18#discussion_r1700085808 - @7171mayanktyagi my comment wasn't realated to cookies maxage, my comments was suggestion to use constants define in class level the value of the constant what would be used for the calculatation. 
Based on Java Documentation suggesting to use the Constants in code instead of numbers, which i suggested, as well as the better practise. Upto you will you use it or no, this is just the suggestion and again I am not the code owner to restrict your commit. I hope I am clear in my explanations

https://github.com/kinde-oss/kinde-java-sdk/pull/18#discussion_r1700091839 - @7171mayanktyagi the [upwork job post](https://www.upwork.com/jobs/Review-for-Java-SDK_~01dbca499b5475fc7e/?referrer_url_path=find_work_home) not visible the company name, because that is job post which you can apply and if the Job poster will respond you then you can see the company/person. I just see on comments the people when adding the feedback the name Andre, not sure if Andre is part of the Kinde.

Hope my clarifications is enough for you.